add user_auth

Spidey03 opened the pull request.

Spidey03 committed:
add user_auth

raviteja1766 commented on user_auth/admin.py:
```diff
@@ -0,0 +1,7 @@
+from django.contrib.auth.admin import UserAdmin
+
+
+admin.site.register(User, UserAdmin)
```
Add new line at the end of the file

raviteja1766 commented on user_auth/presenters/presenter_implementation.py:
```diff
@@ -0,0 +1,28 @@
+    import INVALID_USERNAME, INVALID_PASSWORD, USER_NOT_ADMIN
+from covid_dashboard.interactors.presenters.presenter_interface\
+    import PresenterInterface
+from django_swagger_utils.drf_server.exceptions import NotFound, Forbidden
```
External imports at the top of the file #CleanCode

raviteja1766 commented on user_auth/views/user_login/__init__.py:
```diff
@@ -0,0 +1,2 @@
+API_ENVIRONMENT = "ENV_MOCK"  # ENV_MOCK / ENV_IMPL
```
Change into implementation from mock

Spidey03 committed:
Resolved

Spidey03 committed:
Add Stats details with admin
Add Service Interface in user_authentication
Add Service adapters in covid_dashboard

raviteja1766 commented on ib_miniprojects_backend/settings/.~c9_invoke_7WVLii.py:
```diff
@@ -0,0 +1,349 @@
+DEFAULT_OAUTH_CLIENT_ID = ""
+DEFAULT_OAUTH_CLIENT_SECRET = ""
+DEFAULT_OAUTH_SCOPES = "read write"
+DEFAULT_ACCESS_TOKEN_EXPIRY_IN_SECONDS = 10000000
```
add new online

Spidey03 replied:
I have deleted the file....
It is an unnecessary file

raviteja1766 commented on user_auth/constants/exception_messages.py:
```diff
@@ -0,0 +1,49 @@
+
+USER_NOT_ADMIN = (
+    "you can access this", "USER_NOT_ADMIN"
+)
```
add new line

raviteja1766 reviewed (changes requested):
Change those updates

Spidey03 committed:
resolved pr-1 #3
